doc/facts.rst

```diff
@@ -29,6 +29,8 @@
 - ``jboss.brms.kie-war-ver`` - KIE runtime version
 - ``jboss.eap.common-files`` - Presence of common files and directories for JBoss EAP
 - ``jboss.eap.deploy-dates`` - List of deployment dates of JBoss EAP installations
+- ``jboss.eap.eap-home`` - Find EAP_HOME directories of JBoss EAP installations
+- ``jboss.eap.init-files`` - Find system services that look JBoss-related
 - ``jboss.eap.installed-versions`` - List of installed versions of JBoss EAP
```

Comment:
I wonder if there is some way we can get camayoc to consume what the facts are in rho. It seems a bit looney-tunes that we are keeping these lists in sync manually. 

Granted we want to know some other details about them, but let me know if you have any bright ideas. This is where we store all ours: https://github.com/quipucords/camayoc/blob/master/camayoc/constants.py